config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -110,40 +73,3 @@ periodics:
     testgrid-tab-name: Kubernetes Master Driver Latest
     testgrid-alert-email: gke-managed-storage-alerts@google.com
     description: Kubernetes Integration tests for Kubernetes Master branch, Driver latest build, and Stable Sidecars
-- name: ci-gcp-compute-persistent-disk-csi-driver-latest-k8s-master-integration-canary-sidecars
```

Comment:
The purpose of this job was to potentially catch issues early in the csi sidecars before we actually release them. I think this is particularly useful for csi-attacher, because we can't get good coverage with the hostpath driver.

Reply:
discussed offline, I agree. I put the sidecar canary back in & updated the PR description.